webapp/views.py

```diff
@@ -1,7 +1,4 @@
 # Standard library
-import hashlib
-import hmac
-import json
```

Comment:
I think removing json has broken usn's and cve's?

Reply:
did it? Where did you see that?